### DISCUSSION TITLE
Shortening the timeline

### DISCUSSION THREAD

**Discussion by magnilens** [tomasklaen/uosc#694] [Q&A]
I finally realized what was bothering me about uosc's timeline. The duration displays at either end overlap the timeline. That causes the current position indicator to be a bit hard to see near the start and end of the timeline. 
I've looked through the uosc.conf and I can't find any option that looks like it might shorten the timeline. I also don't know if the duration displays are hard set into their positions in the timeline. In that case, just changing the length of the timeline wouldn't change anything.
So, does anyone know more about this?
I'm not attached to mpv's timeline, I could switch to it, but I prefer staying with uosc as much as possible.

**Comment by tomasklaen** [comment]
That's how uosc is designed, and I don't see an issue here. It might be if you're using a bold by default font that covers the whole height of the timeline, in which case I suggest using something more sensible or maybe use `font_scale` to make it shorter.

Timeline will continue to be the whole width of the window, as that is from design and UX perspective the nicest and most usable, especially in small windows.

**Reply by magnilens** [reply]
I'm not making a feature request. I would like to know if I can customize the timeline in such a way that it doesn't overlap the duration displays anymore. Shortening the timeline seems the simplest to me. But moving the displays away would do as well.

**Reply by tomasklaen** [reply]
No, this kind of customization is not supported.

**Comment by magnilens** [comment]
Oh well. I can live with it.
